prow_config.yaml

```diff
@@ -23,3 +92,62 @@ workflows:
       - components/crud-web-apps/common/frontend/kubeflow-common-lib/*
       - components/crud-web-apps/jupyter/*
     kwargs: {}
+  # Create and push Access Management OCI image to ECR
+  - py_func: kubeflow.kubeflow.cd.access_management.create_workflow
+    name: a-man-build
```

Comment:
lets rename this to `ac-mgr-build`

Reply:
Same here.
I'll need to check the length, but it might be that the 1 extra character will cause the length to be over them max 63 characters.

Reply:
As above, IIUC we should be OK https://github.com/kubeflow/kubeflow/pull/5640#discussion_r589421496